docs/develop/auth.md

```diff
@@ -76,10 +76,11 @@ func AuthHandler(ctx context.Context, token string) (auth.UID, error) {
 }
 ```
 
+## Handling auth errors
 
-## Returning custom error messages on auth handlers
+When a token doesn't match your auth rules (for example if it's expired, the token has been revoked, or the token is invalid), you should return a non-nil error from the auth handler.
 
-When you want to return a custom error message in the auth handler e.g To return an `unauthenticated` error message:
+Encore passes the error message on to the user when you use [Encore's built-in error package](errors), so we recommend using that with the error code `Unauthenticated` to communicate what happened. For example:
 
 ```go
 import (
```

Comment:
I think this code sample can drop the imports and the `// AuthHandler can be named whatever you prefer (but must be exported).` since the previous code samples cover these.

Reply:
Okay

Reply:
`encore.dev/beta/errs` is not imported by other code samples ?